languages/scm.sh

```diff
@@ -121,7 +121,7 @@ compile_payload_scm()
 	if [ -f $scm_hdr ]; then scm_hdr="-e '(load \"$scm_hdr\")'"; else scm_hdr=""; fi
 	gsc_processing=""
 	# if [ $SYS_VERBOSE ]; then gsc_processing="$gsc_processing -expansion"; fi
-        veval "$SYS_GSC -prelude \"$scm_opts\" -c -o $scm_ctgt $gsc_processing $scm_hdr $scm_src"
+        veval "$SYS_GSC -:~~tgtlib=${SYS_PREFIX}/lib -prelude \"$scm_opts\" -c -o $scm_ctgt $gsc_processing $scm_hdr $scm_src"
```

Comment:
`tgtlib` is a new variable you are introducing, right as it doesn't overwrite any internal ones I can see in the gambit source?
Note to self: The syntax for this is documented here: http://www.iro.umontreal.ca/~gambit/doc/gambit.html#Runtime-options